cypress/support/fragments/check-in-actions/checkInActions.js

```diff
@@ -115,5 +117,24 @@ export default {
     cy.wait('@end-patron-session').then(xhr => {
       cy.wrap(xhr.response.statusCode).should('eq', 204);
     });
+  },
+  checkFeeFinesDetails(billedAmount, instanceBarcode, loanPolicyName, OverdueFinePolicyName, LostItemFeePolicyName) {
+    cy.do(availableActionsButton.click());
+    cy.do(feeFineDetailsButton.click());
+    
+    cy.expect(Pane(including('Fee/fine details')).exists());
+    cy.expect(feeFinePane.find(HTML(including('Overdue fine'))).exists());
+    // Using try/catch because the Billed amount is rounded depending on the time and can be equal or greater by 1
+    try {
```

Comment:
you can user OR with interactors instead of try catch block
https://frontside.com/interactors/docs/matchers
example:
```
Link('Google').has({
  href: or(
    'https://google.ca',
    'https://google.co.uk'
  )
});
```